### ISSUE TITLE
Build with electron-packager fails with Error: Cannot find module 'package.json'

### ISSUE THREAD

**Issue by TopCat2** [electron-userland/electron-packager#443]
**Please describe your issue:**

I am getting the message "Error: Cannot find module 'package.json'" in a build for any OS.  package.json is present.  I have read threads about the same message and ensured that the Electron version is set, both in the package.json and, in testing, on the command line with --version.

**Console output when you run electron-packager with the environment variable `DEBUG=electron-packager`. Please include the stack trace if one exists.**

Please NOTE that 'gnome' is the name of my Macbook Air, not the OS that is running.

```
gnome:electron-quick-start Peter$ node_modules/.bin/electron-packager  . --out=dist/osx --platform=darwin --arch=x64
node_modules/.bin/electron-packager  . --out=dist/osx --platform=darwin --arch=x64
  electron-packager Electron Packager 7.5.0 +0ms
  electron-packager Node v6.1.0 +3ms
  electron-packager Host Operating system: darwin (x64) +2ms
  electron-packager Target Platforms: darwin +0ms
  electron-packager Target Architectures: x64 +0ms
module.js:440
    throw err;
    ^

Error: Cannot find module 'package.json'
    at Function.Module._resolveFilename (module.js:438:15)
    at Function.Module._load (module.js:386:25)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at getNameAndVersion (/Users/Peter/hack/electron-quick-start/node_modules/electron-packager/index.js:52:13)
    at packager (/Users/Peter/hack/electron-quick-start/node_modules/electron-packager/index.js:207:3)
    at Object.<anonymous> (/Users/Peter/hack/electron-quick-start/node_modules/electron-packager/cli.js:22:1)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
gnome:electron-quick-start Peter$
```

**What command line arguments are you passing? Alternatively, if you are using the API, what
parameters are you passing to the `packager()` function?**

```
node_modules/.bin/electron-packager  . --out=dist/osx --platform=darwin --arch=x64
```

**Please provide either a failing minimal testcase (with code) or detailed steps to reproduce your
problem. Using [electron-quick-start](https://github.com/electron/electron-quick-start) is a good
starting point.**

Start with electron-quick-start
Add electron-packager
Run npm run pack:osx

My particular failing build is here: https://github.com/TopCat2/ElectronApp

Strangely, I was able to build at one point.  My main branch no longer builds, a checkout of an older version no longer builds, and a fresh copy of electron-quick-start and electron-packager no longer builds.  This leads me to suspect something environmental.  I am running OSC El Capitan, Version 10.11.6

**Comment by TopCat2** [comment]
Installing electron-packager globally, instead of as a dev dependency, allows me to build.  Curious.
I've been working on this for three days.  I just did yet another clone of electron-quick-start and tried the build, which failed.  I then cleaned out node_modules, removed electron-packager and electron-builder from the package.json, did an npm install and tried the build.  It worked.

I went back to my original failed directory and did the same thing.  The package worked.  So either something changed overnight or it behaves differently when installed globally.
